backend/repository/manager.go

```diff
@@ -179,6 +179,11 @@ func (m *Manager) CompileThriftFile(gateway, path string) (*Module, error) {
 	return CompileThriftFile(absPath, prefix)
 }
 
+// CompileThriftCode parses raw bytes in to compiled thrift.
+func (m *Manager) CompileThriftCode(rawCode string) (*Module, error) {
+	return CompileThriftCode([]byte(rawCode), "", "")
```

Comment:
why the wrapper  / why not support include

Reply:
- wrapper: All of the code that uses thrift compilation goes through Manager. Seems like Manager is acting as an abstraction on top of thrift compiler operations. I didn't want to break the abstraction (generally, seems like a decent idea too).
- include support: the test cases does have include `base.thrift` in there and the compilation succeeds. The controller method isn't looking for the included function definition as it isn't a use case we have from for this controller. We can always come back and update the controller.